### ISSUE TITLE
ImportError: cannot import name 'StrEnum' from 'enum' (/usr/lib/python3.10/enum.py)

### ISSUE THREAD

**Issue by yochananmarqos** [alexdelorenzo/cast_control#21]
I'm trying out `cast_control` and `mpris_server` both from the latest commits. I have `StrEnum` 0.4.10 installed, so I'm guessing something changed in Python 3.11? I'm still using 3.10.10.

```
Traceback (most recent call last):
  File "/usr/bin/cast_control", line 5, in <module>
    from cast_control.app.cli import cli
  File "/usr/lib/python3.10/site-packages/cast_control/app/cli.py", line 9, in <module>
    from .daemon import DaemonArgs, MprisDaemon, get_daemon, get_daemon_from_args
  File "/usr/lib/python3.10/site-packages/cast_control/app/daemon.py", line 10, in <module>
    from .state import setup_logging
  File "/usr/lib/python3.10/site-packages/cast_control/app/state.py", line 13, in <module>
    from ..base import DARK_END, DARK_ICON, DATA_DIR, DESKTOP_NAME, DESKTOP_SUFFIX, DESKTOP_TEMPLATE, LIGHT_END, \
  File "/usr/lib/python3.10/site-packages/cast_control/base.py", line 4, in <module>
    from enum import StrEnum, auto
ImportError: cannot import name 'StrEnum' from 'enum' (/usr/lib/python3.10/enum.py)
```

**Comment by damntourists** [comment]
StrEnum is only available in Python 3.11+ from the builtin enum python library:

stdlib/enum.pyi has:
```
if sys.version_info >= (3, 11):
    class StrEnum(str, ReprEnum):
        def __new__(cls, value: str) -> Self: ...
        _value_: str
        @_magic_enum_attr
        def value(self) -> str: ...
        @staticmethod
        def _generate_next_value_(name: str, start: int, count: int, last_values: list[str]) -> str: ...
```

**Comment by yochananmarqos** [comment]
I completely forgot about this issue. It's no longer revelvant.
